panther_driver/README.md

```diff
@@ -9,19 +9,19 @@ Software for controlling Panther robot motors via CAN interface.
   - `/joint_states` *(sensor_msgs/JointState)*
   - `/pose` *(geometry_msgs/Pose)*
   - `/tf` *(tf2_msgs/TFMessage)*
-  - `/odom/wheel` *(nav_msgs/Odometry)* - default not active
+  - `/odom/wheel` *(nav_msgs/Odometry)*
 
 For a `/joint_states` message is crying given data:
 
 - `position = [Front left, Front right, Rear left, Rear right]` - Encoder pulses
 
 - `velocity = [Front left, Front right, Rear left, Rear right]` - Encoder pulses per second
 
-- `effort = [Front left, Front right, Rear left, Rear right]` - Motor current in Amps
+- `effort = [Front left, Front right, Rear left, Rear right]` - Approximate motor current in Nm
```

Comment:
```suggestion
- `effort = [Front left, Front right, Rear left, Rear right]` - Approximate motor torque in Nm
```

Reply:
My bad. For some reason, I made this typo...